source/CommonJobs/CommonJobs.MVC.UI/Content/Evaluations/evaluations.css

```diff
@@ -1299,4 +1299,7 @@ tr.evaluation-generated td {
,opacity 0.5s ease}.modal-dialog:target,.modal-dialog:target .modal-dialog__wrapper,.modal-dialog .modal-dialog--is-open{visibility:visible;opacity:1;-webkit-transition-delay:0;transition-delay:0}.modal-dialog:target .modal-dialog__wrapper{margin-top:0}.modal-dialog__wrapper{max-width:80%;max-height:80%;min-width:37.5em;min-height:34.375em;overflow:auto;position:absolute !important;margin-top:-1.5%;padding:20px;background-color:#fff;border-radius:2px;box-shadow:0 0 1px rgba(0,0,0,0.25);opacity:0;-webkit-transition:visibility 0s ease 0.3s,opacity 0.75s ease 0.4s,margin 0.75s ease 0.4s;transition:visibility 0s ease 0.3s,opacity 0.75s ease 0.4s,margin 0.75s ease 0.4s;position:relative;top:50%;left:50%;-webkit-transform:translate(-50%, -50%);-ms-transform:translate(-50%, -50%);transform:translate(-50%, -50%)}.modal-dialog__close{position:absolute;top:20px;right:20px;text-decoration:none !important;text-transform:uppercase;font-size:12px;font-weight:700;color:#333 !important;opacity:0.5}.modal-dialog__close:hover{opacity:0.8}.modal-dialog__close-target{width:0;height:0;display:block;visibility:hidden;overflow:hidden;position:fixed}.table{width:100%;margin-top:1.25em}.table.filter--is-active-1 th:nth-child(1) a,.table.filter--is-active-1 td:nth-child(1){color:#687392;font-weight:700}.table.filter--is-active-2 th:nth-child(2) a,.table.filter--is-active-2 td:nth-child(2){color:#687392;font-weight:700}.table.filter--is-active-3 th:nth-child(3) a,.table.filter--is-active-3 td:nth-child(3){color:#687392;font-weight:700}.table.filter--is-active-4 th:nth-child(4) a,.table.filter--is-active-4 td:nth-child(4){color:#687392;font-weight:700}.table.filter--is-active-5 th:nth-child(5) a,.table.filter--is-active-5 td:nth-child(5){color:#687392;font-weight:700}.table.filter--is-active-6 th:nth-child(6) a,.table.filter--is-active-6 td:nth-child(6){color:#687392;font-weight:700}.table.filter--is-active-7 th:nth-child(7) a,.table.filter--is-active-7 td:nth-child(7){color:#687392;font-weight:700}.table.filter--is-active-8 th:nth-child(8) a,.table.filter--is-active-8 td:nth-child(8){color:#687392;font-weight:700}.table.filter--is-active-9 th:nth-child(9) a,.table.filter--is-active-9 td:nth-child(9){color:#687392;font-weight:700}.table.filter--is-active-10 th:nth-child(10) a,.table.filter--is-active-10 td:nth-child(10){color:#687392;font-weight:700}.table td,.table th{padding:20px;text-align:left;border-bottom:1px solid #d9d9d9}.table thead th a{font-size:0.875em;font-weight:400;color:#999;text-transform:uppercase}.table thead th a:hover,.table thead th a.filter--is-active{color:#687392}.table thead th a.filter--is-active{font-weight:700}.table thead th a [class*="icon-"]{font-size:80%;vertical-align:baseline}.table tbody td{padding:30px 20px;font-size:0.9375em;color:#999}.status-badge{display:inline-block;margin-right:3px;width:10px;height:10px;border-radius:624.9375em}.status-badge.status--active{background-color:#6ACE9F}.status-badge.status--ended{background-color:#ccc}
```

Comment:
Should this be uncompressed for developers to work on? Where would a developer change values if they need to?